static/templates/user_info_popover_content.hbs

```diff
@@ -150,4 +150,37 @@
             <i class="fa fa-paper-plane" aria-hidden="true"></i> {{#tr this}}View messages sent{{/tr}}
         </a>
     </li>
+    {{#if is_user_admin}}
+        {{#if is_user_owner}}
+            {{#unless is_owner}}
+            <li>
+                {{#if is_active}}
+                <a class="deactivate_user_from_popup_info">
+                    <i class="fa fa-user-times" aria-hidden="true"></i> {{#tr this}}Deactivate user{{/tr}}
+                </a>
+                {{else}}
+                <a class="reactivate_user_from_popup_info">
+                    <i class="fa fa-user-plus" aria-hidden="true"></i> {{#tr this}}Reactivate user{{/tr}}
+                </a>
+                {{/if}}
+            </li>
+            {{/unless}}
+        {{else}}
+            {{#unless is_owner}}
+                {{#unless is_me}}
+                <li>
+                    {{#if is_active}}
+                    <a class="deactivate_user_from_popup_info">
+                        <i class="fa fa-user-times" aria-hidden="true"></i> {{#tr this}}Deactivate user{{/tr}}
+                    </a>
+                    {{else}}
+                    <a class="reactivate_user_from_popup_info">
+                        <i class="fa fa-user-plus" aria-hidden="true"></i> {{#tr this}}Reactivate user{{/tr}}
+                    </a>
+                    {{/if}}
+                </li>
+                {{/unless}}
+            {{/unless}}
+        {{/if}}
+    {{/if}}
```

Comment:
This is a ton of conditional code in the template; we should determine these things in JS.  E.g. probably there's a block of code that could be a function in the settings JS that we could call to do `can_deactivate = can_deactivate(target_user_id)` , and then this can just check that to decide whether to render the "deactivate" widget.